### ISSUE TITLE
A a “download snapshot” button for easier batch transfers / Aidrops on Explorer

### ISSUE THREAD

**Issue by CoreytMayweather** [gochain-io/explorer#219]
I’m on mobile right now so I’ll explain this in layman’s terms.

I put forth an idea to add an “Download Snapshot” button on Go-20 tokens profiles.

The Snapshot is a constantly updating feed of the “TokenName” in each individual address. 

The Snapshot is categorized  like

“0xAddress1” , uint Amount1
“0xAddress2” , uint Amount2
Etc 

Basically an Array .

This would allow for dAPPs to take advantage of easier access, and make autonomous batch transfers a faster Adoption. This would allow more information  to be spread linearly and quicker.

**Comment by jmank88** [comment]
The [API](https://github.com/gochain-io/explorer#get-the-specific-contracts-token-holders-for-go20-compatible-contracts) used by the front end is available to query directly.

For example:
https://explorer.gochain.io/api/address/0x37C93A80b5702A820CbC3050f0e9eC48a1D5e738/holders?limit=5&skip=0
```sh{
   "token_holders":[
      {
         "contract_address":"0x37C93A80b5702A820CbC3050f0e9eC48a1D5e738",
         "token_holder_address":"0x7499bdDF8fEDB3085F305BF755D75A9568fb9255",
         "balance":"999936976338817371000000003",
         "balance_int":999936976,
         "updated_at":"2019-01-23T22:34:51.898Z"
      },
      {
         "contract_address":"0x37C93A80b5702A820CbC3050f0e9eC48a1D5e738",
         "token_holder_address":"0x6767521C414c588bFE00D885AdB2ed0e7b6B9Eb7",
         "balance":"25017000000000000000000",
         "balance_int":25017,
         "updated_at":"2019-01-19T17:48:24.495Z"
      },
      {
         "contract_address":"0x37C93A80b5702A820CbC3050f0e9eC48a1D5e738",
         "token_holder_address":"0x8AeD28c4Ab658Ce5D57D42720d27018207C7C7E9",
         "balance":"8181000000000000000000",
         "balance_int":8181,
         "updated_at":"2019-01-19T17:48:45.073Z"
      },
      {
         "contract_address":"0x37C93A80b5702A820CbC3050f0e9eC48a1D5e738",
         "token_holder_address":"0x2893b6fF32F9E59be6790918F8a52740a47E588B",
         "balance":"2725000000000000000000",
         "balance_int":2725,
         "updated_at":"2019-01-19T17:48:28.723Z"
      },
      {
         "contract_address":"0x37C93A80b5702A820CbC3050f0e9eC48a1D5e738",
         "token_holder_address":"0xd4E4C1B68367A6A02A0Db7eDf4eEdD84A80d2EF6",
         "balance":"2552000000000000000000",
         "balance_int":2552,
         "updated_at":"2019-01-19T17:48:50.626Z"
      }
   ]
}
```
Is this what you are looking for?

**Comment by CoreytMayweather** [comment]
Yes!! I’d have build my own explorer?

**Comment by jmank88** [comment]
I'm not sure I understand. This particular data is already featured here: https://explorer.gochain.io/addr/0x37C93A80b5702A820CbC3050f0e9eC48a1D5e738?addr_tab=holders

**Comment by CoreytMayweather** [comment]
Where is the direct API to access it

**Comment by jmank88** [comment]
Linked from my (previous comment)[https://github.com/gochain-io/explorer/issues/219#issuecomment-457058082]
